### PR TITLE
Refactor for early return

### PR DESCRIPTION
**Pull request by drbyte**
Easier to see the identical patterns when debugging.
In the case of the payment class it makes the code much more like an interface, which could be integrated later particularly by de-duplicating etc.

### CODE REVIEW

File: includes/classes/payment.php [hunk 1]
```diff
@@ -92,30 +92,28 @@ function __construct($module = '') {
...
   function update_status() {
-    if (empty($this->selected_module)) return; 
-    if (is_array($this->modules)) {
-      if (is_object($GLOBALS[$this->selected_module])) {
-        if (method_exists($GLOBALS[$this->selected_module], 'update_status')) {
-          $GLOBALS[$this->selected_module]->update_status();
-        }
-      }
-    }
+    if (empty($this->selected_module)) return;
```
Comment (lat9) [inline includes/classes/payment.php:111]: Any reason for multiple `if` statements instead of aggregating on a single line?

Comment (lat9) [inline includes/classes/payment.php:111]: Any reason for the multi-if-statements instead of aggregating on a single `if`?

Comment (drbyte) [reply includes/classes/payment.php:111]: Largely easier readability.

Comment (drbyte) [reply includes/classes/payment.php:111]: To clarify: the "early return" concept allows one to quickly see all the criteria that do not apply to the outcome, and then see only the actual real "actions" once all those disqualifiers are out of the way.
Several short `if` statements that just directly return/abort makes human reading of it very quick.

Comment (lat9) [reply includes/classes/payment.php:111]: Different strokes for different folks.

**Pull request merged**